tests/app/main/views/test_accept_invite.py

```diff
@@ -335,7 +337,9 @@ def test_existing_user_of_service_get_redirected_to_signin(
         page.select("main p")[0].text.strip(),
     ) == (
         "You need to sign in again",
-        "We signed you out because you have not used Notify for a while.",
+        # TODO:  Improve this given Login.gov configuration.
+        # "We signed you out because you have not used Notify for a while.",
+        "You have  left to use Login.gov to sign in",
```

Comment:
looks like two spaces to the left of 'left'.

Reply:
There are, yes - this is the exact output from the tests so I'm going with the text verbatim for now.  Hence the `TODO` note!